projects/canopy/src/lib/table/table/table.component.spec.ts

```diff
@@ -84,12 +84,23 @@ describe('TableComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should have the default class', () => {
+  it('should have the default classes', () => {
     expect(tableDebugElement.nativeElement.getAttribute('class')).toBe(
-      'lg-table--columns-md lg-table',
+      'lg-table--striped lg-table--columns-md lg-table',
     );
   });
 
+  describe('when a variant is specified', () => {
+    it('should set the correct class modifier', () => {
+      component.variant = 'bordered';
+      fixture.detectChanges();
+
+      expect(tableDebugElement.nativeElement.getAttribute('class')).toContain(
+        'lg-table--bordered',
+      );
```

Comment:
Perhaps we should also test here that the doesn't have the `lg-table--striped` class, as there could be a risk that the component logic fails to remove one while adding another. The could go for the `it('should have the default classes')` test above, perhaps that check that the bordered class is not present.

Reply:
Done